pom.xml

```diff
@@ -43,7 +43,7 @@
     </issueManagement>
     <ciManagement>
         <system>Travis CI</system>
-        <url>https://travis-ci.org/PicnicSupermarket/oss-parent</url>
+        <url>https://travis-ci.com/PicnicSupermarket/oss-parent</url>
```

Comment:
We first needed to sign up for the beta and trigger a migration. Did that now.

Reply:
Thx! I missed that part.